src/Microsoft.IdentityModel.Logging/IdentityModelEventSource.cs

```diff
@@ -265,7 +265,7 @@ public void WriteCritical(string message, params object[] args)
         [NonEvent]
         public void Write(EventLevel level, Exception innerException, string message)
         {
-            Write(level, innerException, message, null);
+            Write(level, innerException, message, null!);
```

Comment:
`null!` hints that something isn't right here.

Why did we end up with this solution?

Reply:
I'd think that you would need:
`public void Write(EventLevel level, Exception innerException, string message, params object[]? args)` 
and then 
`public void WriteAlways(string message, params object[]? args)`

which already has a null check for null `args`

Reply:
Great catch, I muted off this error because I started working on a different as it was very repetitive. I'll update this.

Reply:
Can you resolve the comments which are fixed? thank you